core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitorSpec.scala

```diff
@@ -830,6 +848,43 @@ class AvroUpsertMonitorSpec(_system: ActorSystem)
 
   }
 
+  it should "ack pubsub message if message is double delivered" in withTestDataApiServices { services =>
+    val timeout = 30000 milliseconds
+    val interval = 250 milliseconds
+    val importUuid = UUID.randomUUID()
+
+    setUpMockImportServiceToReturnStatus(services, ImportStatuses.Done)
```

Comment:
hm - it works for me:
```suggestion
    val mockImportServiceDAO = setUpMockImportService(services)
    when(
      mockImportServiceDAO.getImportStatus(any[UUID], any[WorkspaceName], any[UserInfo])
    ).thenReturn(Future.successful(Some(ImportStatuses.Done)))
```

Reply:
Huh, I must have been doing something silly.  When I let the IDE take care of this for me it worked fine and was easy to modify to produce what you provided.  Fixed in 0bd38c139e83f5050c6c471b75df87313bcc6572